autopush/web/base.py

```diff
@@ -56,22 +59,33 @@ def _validate_request(self, request_handler):
         schema.context["log"] = self.log
         return schema.load(data)
 
-    def _call_func(self, result, func, request_handler, *args, **kwargs):
+    def _call_func(self, result, func, request_handler, start_time, *args,
```

Comment:
forgot this: _track_validation_timing will always track the thread's timing so _call_func doesn't need to do it any longer